This patch removes the dependency on node types
Can be removed once https://github.com/software-mansion/react-native-gesture-handler/pull/1420 is released

diff --git a/node_modules/react-native-gesture-handler/lib/typescript/components/touchables/GenericTouchable.d.ts b/node_modules/react-native-gesture-handler/lib/typescript/components/touchables/GenericTouchable.d.ts
index 01aa75f..4ef8307 100644
--- a/node_modules/react-native-gesture-handler/lib/typescript/components/touchables/GenericTouchable.d.ts
+++ b/node_modules/react-native-gesture-handler/lib/typescript/components/touchables/GenericTouchable.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="node" />
 import { Component } from 'react';
 import { StyleProp, ViewStyle, TouchableWithoutFeedbackProps } from 'react-native';
 import { GestureEvent, HandlerStateChangeEvent } from '../../handlers/gestureHandlers';